specs/anchored-region/anchored-region.md

```diff
@@ -0,0 +1,260 @@
+# Anchored region
+
+## Overview
+
+An *anchored region* is a container component which enables authors to create layouts where the contents of the anchored region can be positioned relative to another "anchor" element.  Additionally, the *anchored region* can react to the available space between the anchor and a parent ["viewport"](https://developer.mozilla.org/en-US/docs/Glossary/viewport) element such that the region is placed on the side of the anchor with the most available space, or even resize itself based on that space.
+
+### Background
+
+This component is inspired by the ["Viewport positioner"](https://github.com/microsoft/fast-dna/issues/2597)  component in the React component set.  It is used as a building block in other components to enable responsive flyouts, or positionable/scaling menus in the [select](https://explore.fast.design/components/select) component. 
```

Comment:
I would not link to select in the explore site as that link may break in the future as we prepare to launch the web components, instead could you refer to selects and context menus as a general group of components?

Reply:
Pointing to the code file for React Select rather than the explore site. I think as "background" pointing to previous implementations make sense as a tecnical reference.